compiler/rustc_metadata/src/rmeta/encoder.rs

```diff
@@ -599,6 +599,7 @@ impl<'a, 'tcx> EncodeContext<'a, 'tcx> {
         macro_rules! stat {
             ($label:literal, $f:expr) => {{
                 let orig_pos = self.position();
+                $label.encode(&mut self.opaque);
```

Comment:
Maybe `format!("<<{}>>", $label).encode(&mut self.opaque)` to make it a bit clearer that it is a label rather than just a random string?